main.go

```diff
@@ -130,15 +142,9 @@ func getJoinYipStages(cluster clusterplugin.Cluster) []yip.Stage {
 	}
 }
 
-func getInitNodeConfiguration(cluster clusterplugin.Cluster) string {
+func getInitNodeConfiguration(cluster clusterplugin.Cluster, initCfg kubeadmapiv3.InitConfiguration, clusterCfg kubeadmapiv3.ClusterConfiguration) string {
 	certificateKey := getCertificateKey(cluster.ClusterToken)
 
-	var initCfg kubeadmapiv1.InitConfiguration
-	var clusterCfg kubeadmapiv1.ClusterConfiguration
-
-	_ = yaml.Unmarshal([]byte(cluster.Options), &initCfg)
-	_ = yaml.Unmarshal([]byte(cluster.Options), &clusterCfg)
-
 	substrs := bootstraputil.BootstrapTokenRegexp.FindStringSubmatch(cluster.ClusterToken)
```

Comment:
what's the clusterToken here? I remember for rke2/k3s, this is just the clusterId, will that work? or this will be different?

Reply:
Just the cluster id will not work here. The token for kubeadm requires a specific format. There will be a change on the stylus side to turn the cluster id into that format.  
I am using the same logic as was done for stylus v1: https://github.com/spectrocloud/stylus/blob/ac9d7fa856783914a162dfdd644b7c0c7a4b6cb0/cli/deploy/import.go#L233